tex/chapters/07-ApplyingModel.tex

```diff
@@ -0,0 +1,197 @@
+%!TEX root = ../main.tex
+
+\chapter{Application of the Model}\label{cha:applying-model}
+
+In this Chapter a method for assessing the sensitivity of the model will be outlined.
+This is done by making several small changes to the parameters of strategies and observing whether the model can discriminate between the new altered vectors.
+The size of the change made to the strategy will be varied and the models performance recorded for each deviation.
+It is expected that there is a cut-off where the model changes from performing well to performing badly, and this gives an approximation of it's sensitivity.
+Finally, model will be applied to the Axelrod-Python library to determine if it contains any duplicate strategies.
+
+
+
+\section{MemoryOne Strategies}
+
+A MemoryOne strategy bases its decision entirely on the the last round of play.
+In Axelrod-Python they are defined by a vector $v$ of length 4 that determines the probability of Cooperation given the previous rounds history, ie:
+$$
+v = (P(C|CC), P(C|CD), P(C|DC), P(C|DD))
+$$
+For example, a MemoryOne strategy defined by the vector $v=(1, 1, 1, 1)$ is equivalent to Cooperator.
+TitForTat can be implemented as a MemoryOne player with the vector $v=(1, 0, 1, 0)$, Pavlov as the MemoryOne player with $v=(1, 0, 0, 1)$ and Random as the MemoryOne player with $v=(0.5, 0.5, 0.5, 0.5)$.
+See Section \ref{sec:individual_strategies} for explanations of how these strategies operate.
+
+
+
+\section{Determining the sensitivity of the Model}
+
+All possible MemoryOne strategies are defined by the space $S = [0, 1]^4_{\mathbb{R}}$.
+For a vector $s \in S$ and a deviation $\delta \in (0, 1)$, 16 new deviated vectors are created by taking the sum of $s$ with all possible vectors $(\pm \delta, \pm \delta, \pm \delta, \pm \delta)$, but bounding each element in the range $[0, 1]$.
+For example, with $s = (0.2, 0, 0.5, 0.95)$ and $\delta = 0.1$, the 16 vectors produced are:
+
+\begin{itemize}
+\begin{multicols}{4}
+  \item $(0.3, 0.1, 0.6, 1.0)$
+  \item $(0.3, 0.1, 0.6, 0.85)$
+  \item $(0.3, 0.1, 0.4, 1.0)$
+  \item $(0.3, 0.1, 0.4, 0.85)$
+  \item $(0.3, 0.0, 0.6, 1.0)$
+  \item $(0.3, 0.0, 0.6, 0.85)$
+  \item $(0.3, 0.0, 0.4, 1.0)$
+  \item $(0.3, 0.0, 0.4, 0.85)$
+  \item $(0.1, 0.1, 0.6, 1.0)$
+  \item $(0.1, 0.1, 0.6, 0.85)$
+  \item $(0.1, 0.1, 0.4, 1.0)$
+  \item $(0.1, 0.1, 0.4, 0.85)$
+  \item $(0.1, 0.0, 0.6, 1.0)$
+  \item $(0.1, 0.0, 0.6, 0.85)$
+  \item $(0.1, 0.0, 0.4, 1.0)$
+  \item $(0.1, 0.0, 0.4, 0.85)$
+\end{multicols}
+\end{itemize}
+
+The 16 deviated vectors are then entered into an Ashlock tournament to produce the corresponding results table where all strategies are compared (as described in Section \ref{sec:ashlock_tourn}).
+The model is then be applied to this set of results, and the proportion of comparisons predicted correctly is recorded.
```

Comment:
`The model obtained in ... is then use on this strategies to identify the proportion of correct predictions`.